opensearch-transport/Gemfile

```diff
@@ -37,6 +37,9 @@ if File.exist? File.expand_path('../opensearch/opensearch.gemspec', __dir__)
   gem 'opensearch-ruby', path: File.expand_path('../opensearch', __dir__), require: false
 end
 
+ENV['FARADAY_VERSION'] = '~> 1.0' if RUBY_VERSION.to_f < 2.7
```

Comment:
Btw, `to_f` doesn't work well all the time. Compare versions with `Gem::Version.new()`. Can fix this later.